### ISSUE TITLE
Enhance a11y for `NativeTable`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5395]
name: NativeTable

### Description
3 accessibility violations were found in the `NativeTable`.

#### [Components / NativeTable / ScrollArea](https://yamada-ui.github.io/yamada-ui/?path=/story/components-nativetable--scroll-area)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / NativeTable / Sticky](https://yamada-ui.github.io/yamada-ui/?path=/story/components-nativetable--sticky)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / NativeTable / Sticky](https://yamada-ui.github.io/yamada-ui/?path=/story/components-nativetable--sticky)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
